### PR TITLE
UX: Add custom section button should not be shown to anon users

### PR DESCRIPTION
**Pull request by tgxworld**

### CODE REVIEW

File: app/assets/javascripts/discourse/app/components/sidebar/footer.hbs [hunk 1]
```diff
@@ -3,12 +3,14 @@
     <div class="sidebar-footer-actions">
       <PluginOutlet @name="sidebar-footer-actions" />
 
-      <DButton
-        @icon="plus"
-        @action={{action this.addSection}}
-        @class="btn-flat add-section"
-        @title="sidebar.sections.custom.add"
-      />
+      {{#if this.currentUser}}
```
Comment (tgxworld) [inline app/assets/javascripts/discourse/app/components/sidebar/footer.hbs:6]: This line is the fix

File: spec/system/custom_sidebar_sections_spec.rb [hunk 2]
```diff
@@ -7,9 +7,13 @@
   let(:sidebar) { PageObjects::Components::Sidebar.new }
 
   it "allows the user to create custom section" do
+    visit("/latest")
+
+    expect(sidebar).to have_no_add_section_button
```
Comment (tgxworld) [inline spec/system/custom_sidebar_sections_spec.rb:12]: Tagging on to an existing test so to reduce testing overhead for a small feature

**Review by Drenmi — APPROVED** [review]
🚀

**Comment by tgxworld** [comment]
Thank you for the review @Drenmi

**Pull request merged**